RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -257,3 +269,44 @@ extension View {
 }
 
 #endif
+
+#if DEBUG
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct RestorePurchasesAlert_Previews: PreviewProvider {
+    static var previews: some View {
+        PreviewContainer(alertType: RestorePurchasesAlert.AlertType.restorePurchases)
+            .previewDisplayName("Restore Purchases")
+
+        PreviewContainer(alertType: RestorePurchasesAlert.AlertType.purchasesRecovered)
+            .previewDisplayName("Purchases Recovered")
+
+        PreviewContainer(alertType: RestorePurchasesAlert.AlertType.purchasesNotFound)
+            .previewDisplayName("Purchases Not Found")
+    }
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+private struct PreviewContainer: View {
+    @State private var isPresented = true
+    let alertType: RestorePurchasesAlert.AlertType
+
+    var body: some View {
+        Color.clear
+            .modifier(RestorePurchasesAlert(isPresented: $isPresented, alertType: alertType))
+            .environmentObject(CustomerCenterViewModel(actionWrapper: CustomerCenterActionWrapper()))
+            .environment(\.localization, CustomerCenterConfigTestData.customerCenterData.localization)
+            .emergeRenderingMode(.window)
+            .onAppear {
+                DispatchQueue.main.async {
+                    self.isPresented = true
+                }
```

Comment:
Is the dispatch needed? 🤔

Reply:
oh just setting the `isPresented` you mean? without dispatching?